Assets/MixedRealityToolkit/_Core/Definitions/Devices/DeviceInputType.cs

```diff
@@ -29,6 +29,9 @@ public enum DeviceInputType
         ButtonPress,
         ButtonTouch,
         ButtonNearTouch,
+        SecondaryButtonPress,
```

Comment:
We don't need this. 

A button press is a button press. You're going to have to figure out another way to distinguish.

We want this enum to be atomic.

Reply:
Then come up with a suggestion.  Rift controllers have two buttons on each controller, they don't have an identifying ID other than their input index and we can't have duplicate DeviceInputTypes.
This works for now and we can review again for beta.

Reply:
Should have an id via button press, correct? Or are the button defined by axis?

Reply:
Put it this way, look at the Controller Interaction configuration, with the three drop downs and find a way to configure 4 different buttons with a single "Button" value.

Reply:
Will do

Reply:
We should probably add an enum with valid button press enums to assign.

We also need to clean up that config screen to be more user friendly. I saw your other issue that identifies this.

I'll submit a PR against your branch to add the input button enum option for identifying the input button enum to map against.